src/main.rs

```diff
@@ -10,37 +11,48 @@ struct Cli {
 
 #[derive(Subcommand)]
 enum Commands {
-    /// Adds a package to the project
+    /// Adds packages to the project
     Add {
-        /// Name of the package to add
-        package_name: String,
+        /// Packages in the format package@version
+        packages: Vec<String>,
     },
-    /// Removes a package from the project
+    /// Removes packages from the project
     Remove {
-        /// Name of the package to remove
-        package_name: String,
+        /// Names of the packages to remove
+        package_names: Vec<String>,
     },
 }
 
 fn main() {
     let cli = Cli::parse();
 
     match &cli.command {
-        Commands::Add { package_name } => {
-            add_package(package_name);
+        Commands::Add { packages } => {
```

Comment:
Let's add here a functionality to resolve "latest" package version from if no version is specified. This works for every other package manager. 

1. For `noir-libs add package@<version>` - it stays at is now
2. For `noir-libs add package` - you download latest version.  How to download latest version.

2.1 API will have endpoint to return you a JSON with package details: `GET /api/v1/packages/<package_name>`
Use this to fetch details about package. JSON will include `versions` string array, sorted as "newest versions first". 
2.2 Use latest version to download package. `GET /api/v1/packages/<package_name>/<package_version>/download`